vaadin-crud-flow-parent/vaadin-crud-flow-integration-tests/src/test/java/com/vaadin/flow/component/crud/test/EditorButtonsIT.java

```diff
@@ -0,0 +1,147 @@
+package com.vaadin.flow.component.crud.test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+
+import com.vaadin.flow.component.button.testbench.ButtonElement;
+import com.vaadin.flow.component.crud.testbench.CrudElement;
+import com.vaadin.flow.component.textfield.testbench.TextFieldElement;
+
+public class EditorButtonsIT extends AbstractParallelTest {
+
+    @Before
+    public void init() {
+        String url = getBaseURL().replace(super.getBaseURL(),
+                super.getBaseURL() + "/vaadin-crud") + "/editorbuttons";
+        getDriver().get(url);
```

Comment:
Could use the `@TestPath` annotation + `open()` instead?

Reply:
I think this could be possible if tests were inherited from `AbstractComponentIT` like date-picker ITs.

Reply:
Seems like the other tests use the same setup